src/Core/Form/FormChoiceFormatter.php

```diff
@@ -50,13 +50,25 @@ class FormChoiceFormatter
      */
     public static function formatFormChoices(array $rawChoices, string $idKey, string $nameKey, bool $sortByName = true): array
     {
+        // Remove items with duplicate name and ID
```

Comment:
Is there a test that covers this use case ? 

Can be added later, but this will make the PR perfect :p

Reply:
I will add it in another PR, but good idea. :-) Maybe I will discover more!